RootOutputer.cc

```diff
@@ -20,6 +21,7 @@ RootOutputer::RootOutputer(std::string const& iFileName, unsigned int iNLanes, C
   basketSize_{iConfig.basketSize_},
   splitLevel_{iConfig.splitLevel_}
 {
+  TFileCacheWrite(&file_, iConfig.cacheSize_);
```

Comment:
This probably has to be 
```suggestion
  new TFileCacheWrite(&file_, iConfig.cacheSize_);
```